DCOS/dcos-testing.sh

```diff
@@ -262,7 +262,13 @@ open_dcos_port() {
 }
 
 setup_remote_winrm_client() {
-    upload_files_via_scp -i $PRIVATE_SSH_KEY_PATH -u $LINUX_ADMIN -h $MASTER_PUBLIC_ADDRESS -p "2200" -f "/tmp/wsmancmd" "$DIR/utils/bin/wsmancmd" || {
+    local WSMANCMD_URL="http://dcos-win.westus.cloudapp.azure.com/downloads/wsmancmd"
+    curl --retry 30 "${WSMANCMD_URL}" -o wsmancmd || {
```

Comment:
I know that this *should* execute in the `$WORKSPACE` (unless nobody changed the working directory before calling this), but I suggest using the absolute path: `$WORKSPACE/wsmancmd`.

Reply:
Also, please add `-s` (silent) parameter. It makes curl show an error message if it fails, otherwise, it will silently finish.